twill-yarn/src/main/java/org/apache/twill/internal/appmaster/RunningContainers.java

```diff
@@ -132,16 +149,22 @@ void start(String runnableName, ContainerInfo containerInfo, TwillContainerLaunc
     try {
       int instanceId = getStartInstanceId(runnableName);
       RunId runId = getRunId(runnableName, instanceId);
+      Map<String, LogEntry.Level> logLevelsRunnable = resourceReport.getLogLevels().containsKey(runnableName) ?
+        resourceReport.getLogLevels().get(runnableName) : new HashMap<String, LogEntry.Level>();
+      logLevel = logLevelsRunnable.containsKey(Logger.ROOT_LOGGER_NAME) ?
```

Comment:
Is `ROOT_LOGGER_NAME` a special logger name? If that's the case, we should be consistently using the map only. I see some other classes having both `rootLogLevel` and the map.